SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,74 @@
+  - Assign user stories to the sprint and split user stories into tasks
+  - Set up Work in Progress (WIP) limits
+- [ ] [Decide on test frameworks and discuss test strategies](test-first-development/readme.md)
+  - Discuss the purpose and goals of tests and how to measure test coverage
+  - Agree on how to separate unit tests from integration, load and smoke tests
+  - Design the first test cases
+- [ ] [Decide on branch naming](source-control/feature-branching/readme.md)
+- [ ] [Discuss security needs and verify that secrets are kept out of source control](continuous-deployment/secrets-management/recipes/azure-devops/secrets-per-branch.md)
 
-This workflow makes the following assumptions about the development environment
+### Day 2
 
-- Team is using git for version control.
+- [ ] [Set up Source Control](source-control/readme.md)
+  - Agree on [best practices for commits](source-control/readme.md#commit-best-practices)
+- [ ] [Set up basic Continuous Integration with linters and automated tests](continuous-integration/readme.md)
+- [ ] [Set up meetings for Daily Standups and decide on a scrum master](stand-ups/readme.md)
+  - Discuss purpose, goals, participants and facilitation guidance
+  - Discuss timing, and how to run an efficient stand-up
+- [ ] [If the project has sub-teams, set up a Scrum of Scrums](scrum-of-scrums/readme.md)
+
+### Day 3
+
+- [ ] [Agree on code style](code-reviews/README.md) and on [how to assign Pull Requests](code-reviews/pull-requests.md)
+- [ ] [Set up Build Validation for Pull Requests (2 reviewers, linters, automated tests)](code-reviews/README.md) and agree on [Definition of Done](team-agreements/definition-of-done/readme.md)
+- [ ] [Agree on a Code Merging strategy](source-control/git.md) and update the [CONTRIBUTING.md](resources/templates/CONTRIBUTING.md)
+- [ ] [Agree on logging and observability frameworks and strategies](observability/readme.md)
+
+### Day 4
+
+- [ ] [Set up Continuous Deployment](continuous-deployment/readme.md)
+  - Determine what environments are appropriate for this solution
+  - For each environment discuss purpose, when deployment should trigger, pre-deployment approvers, sing-off for promotion.
+- [ ] [Decide on a versioning strategy](source-control/versioning/readme.md)
+- [ ] Agree on how to [Desing a feature and conduct a Design Review](design-reviews/readme.md)
+
+### Day 5
+
+- [ ] Conduct a Sprint Demo
+- [ ] [Conduct a Retrospective](retrospectives/readme.md)
+  - Determine required participants, how to capture input (tools) and outcome
+  - Set a timeline, and discuss facilitation, meeting structure etc.
+- [ ] [Groom the Backlog](backlog-management/grooming/readme.md)
+  - Determine required participants
+  - Update the [Definition of Ready](team-agreements/definition-of-ready/readme.md)
+  - Update estimates, and the [Estimation](sprint-planning/estimation/readme.md) document
+- [ ] [Submit Engineering Feedback for issues encountered](engineering-feedback/readme.md)
```

Comment:
I think Sprint planning is left out of this convo. For me, it is a process built after backlog grooming and estimation. Also a team agreement of what is committed for the end of the sprint. Thoughts?

Reply:
Sprint planning is covered on day 1. 

Since this is a cycle normally it is hard to find a perfect place for everything, but now building the backlog is done first, followed by a sprint plan and then grooming comes after the first sprint.  I think that feels like a good compromise 

The team commitment is covered IMHO by the sentence under "Plan the first sprint" - "Agree on a sprint goal, and how to measure the sprint progress" - let me know if you disagree and how you would have done it differently

Reply:
Agree with thinking that it is hard to table everything for week 1